terraform/app/modules/cloudfront/variables.tf

```diff
@@ -47,4 +47,10 @@ locals {
   cloudfront_aliases_cnames                              = [for zone in var.route53_zones : "${var.route53_cname_record}.${zone}"]
   cloudfront_aliases                                     = concat(var.route53_a_records, local.cloudfront_aliases_cnames)
   cloudfront_viewer_certificate_minimum_protocol_version = "TLSv1.2_2018"
+  cloudfront_path_pattern                                = { pattern_1 = { path = "/packs/*" }, pattern_2 = { path = "/attachment*" } }
+  cloudfront_custom_response = { response_1 = { er_code = "404", ttl = "10", response_code = "404", page_path = "${var.offline_bucket_origin_path}/index.html" },
+    response_2 = { er_code = "500", ttl = "60", response_code = "500", page_path = "${var.offline_bucket_origin_path}/index.html" },
```

Comment:
`"${var.offline_bucket_origin_path}/index.html"` is always the same, so it could be removed from the map and hard coded in the dynamic block